README.md

```diff
@@ -347,6 +347,20 @@ To avoid storage charges for retaining the temporary composite objects, consider
 ### Google Cloud Storage Class
 Due to the quick creation and deletion of composite objects, we recommend that only the [Standard storage class](https://cloud.google.com/storage/docs/storage-classes) is applied to your bucket to minimize cost and maximize performance.
 
+### Throughput and QPS Limits
+Many machine learning efforts opt for a highly distributed training model leveraging tools such as Pytorch Lightning and Ray. These models are compatible with Dataflux, but can often trigger the rate limits of Google Cloud Storage. This typically manifest in a 429 error, or slower than expected speeds while running distributed operations. Details on specific quotas and limits can be found [here](https://cloud.google.com/storage/quotas).
+
+#### Egress Throughput Limits
+429 errors acompanied with messages indicating `This workload is drawing too much egress bandwidth from Google Cloud Storage` or `triggered the Cloud Storage Egress Bandwidth Cap` indicate that the data throughput rate of your workload is exceeding the maximum capacity of your Google Cloud Project. To address these issues, you can take the following steps:
+
+1. Ensure that object composition is [disabled](#composite-objects) for dataflux download operations.  
+2. Check that other workloads executing within your project are not drawing excess bandwidth.
+3. Lower the scale of the workload in question to reduce its egress bandwidth (e.g. fewer running pods).
```

Comment:
I'm a little hesitant on whether it makes sense to suggest customers to "lower the scale" to work around those problems. I mean, it could make sense to do that to validate the problem but as a solution, I'm not sure?